### PR TITLE
Do not return counts in queryDispatch

### PR DESCRIPTION
**Pull request by aprokop**
We never use it. And if one wants it, could do it through callback.

### CODE REVIEW

File: src/details/ArborX_DetailsTreeTraversal.hpp [hunk 1]
```diff
@@ -33,42 +33,38 @@ struct TreeTraversal
...
 
     if (bvh.size() == 1)
     {
       if (predicate(bvh.getBoundingVolume(bvh.getRoot())))
       {
         insert(0);
-        return 1;
       }
-      else
-        return 0;
```
Comment (dalg24) [inline src/details/ArborX_DetailsTreeTraversal.hpp:65]: Intent was to return early here

Comment (aprokop) [reply src/details/ArborX_DetailsTreeTraversal.hpp:65]: Yeah, my bad.

**Comment by aprokop** [comment]
Retest this please

**Comment by dalg24** [comment]
> Retest this please

It does not compile.

https://github.com/arborx/ArborX/blob/b9b34f7a6b1dc500f3d38aa419cb2abd669d6019/src/details/ArborX_DetailsTreeVisualization.hpp#L223

Also I mean it, you are wasting time with this PR I won't let it through

**Comment by aprokop** [comment]
If we consider everything-is-done-through-callback, then this interface is wrong.

**Pull request closed without merging**